### PR TITLE
Add the open source project named surging

### PR DESCRIPTION
**Pull request by fanliang11**
surging is based on .net core distributed micro-service framework, providing high-performance RPC communication, service registration discovery, service fault tolerance degradation, middleware, API gateway, can effectively improve the system availability and reliability

### CODE REVIEW

**Comment by fanliang11** [comment]
https://github.com/dotnetcore/surging

File: dotnet-developer-projects.md [hunk 1]
```diff
@@ -92,6 +92,7 @@ Please sort projects alphabetically and provide a one-line description. GitHub/C
   * [MassTransit](https://github.com/MassTransit/MassTransit) - A distributed application framework for RabbitMQ and Azure Service Bus.
   * [NServiceBus](https://github.com/Particular/NServiceBus) - The most popular service bus for .NET
   * [RestBus](https://github.com/tenor/RestBus) - Easy Asynchronous Messaging and Queueing for .NET
+  * [surging](https://github.com/dotnetcore/surging) -The distributed micro service framework based on.NET Core provides high-performance RPC Communications
```
Comment (rpetrusha) [inline dotnet-developer-projects.md]: on.NET --> on .NET

Comment (fanliang11) [reply dotnet-developer-projects.md]: Thank you for reviews, changes to Libraries

**Comment by rpetrusha** [comment]
Thanks, @fanliang11. We'll merge your PR now.

**Pull request merged**